conda_build_all/tests/unit/test_version_matrix.py

```diff
@@ -214,6 +214,28 @@ def test_numpy_xx_and_restrictive_specifcation(self):
         self.assertEqual(set(r), set(expect_result),
                          msg='got: {}\nexpected: {}'.format(r, expect_result))
 
+    def test_vc(self):
+        a = DummyPackage('pkgA', ['vc'])
+        self.index.add_pkg('vc', '9')
+        self.index.add_pkg('vc', '10')
+        r = special_case_version_matrix(a, self.index)
+        self.assertEqual(r, set([(('vc', '9'),), (('vc', '10'),),
+                                ])
+                         )
+
+    def test_vc_python(self):
+        a = DummyPackage('pkgA', ['vc', 'python'])
+        self.index.add_pkg('vc', '9')
+        self.index.add_pkg('vc', '10')
+        self.index.add_pkg('python', '2.7', 'py27vc9', depends=['vc 9'])
+        self.index.add_pkg('python', '2.7', 'py27vc10', depends=['vc 10'])
+        self.index.add_pkg('python', '3.4', 'py34', depends=['vc 10'])
+        r = special_case_version_matrix(a, self.index)
+        expected = {(('vc', '9'), ('python', '2.7')),
+                    (('vc', '10'), ('python', '2.7')),
+                    (('vc', '10'), ('python', '3.4'))}
+        self.assertEqual(r, expected)
```

Comment:
The numbers have no significance in these unit tests. For instance, if there is a ``vc 99`` that should also be included.